rundeck-scripts/start-node

```diff
@@ -167,6 +167,13 @@ while (( i )); do
 	if [[ -n "$container_id" ]]; then
 		echo "RNode is running"
 		nohup docker logs -f $container_id &> $DIAG_DIR/console.log &
+
+                node_pid="$(docker inspect -f '{{.State.Pid}}' rnode || echo 0)"
+                if (( node_pid == 0 )); then
+                        echo "Could not get node PID"
+                        break
+                fi
+                nohup ./pmap.py "$node_pid" "perf-bootstrap.c.developer-222401.internal" 8091 testnet1 &
```

Comment:
It's because the file is based on tabs and not spaces.  Was that the intention or a editor misconfiguration?

Reply:
Tabs are my default.

Reply:
Using tabs makes the code appear differently to the reader than to the writer because it depends on the reader's editor configuration.   I think spaces should be preferred everywhere.